Packs/Base/Scripts/CommonServerPowerShell/script-CommonServerPowerShell.yml

```diff
@@ -0,0 +1,735 @@
+commonfields:
```

Comment:
@GalRabin I think you added this file by mistake

Reply:
Right

Reply:
Right sorry for that.